Jenkinsfile

```diff
@@ -106,6 +106,8 @@ try { // Use a try block to perform cleanup in a finally block when the build fa
       } else {
         sh "oc process -f ose3/pipeline-application-template.json -n ${project} | oc apply -f - -n ${project}"
       }
+      // sleep a bit to allow the imagestreams to be populated before we move on to building
+      sh "sleep 30"
```

Comment:
i'd rather see this be a polling loop or some other more intelligent wait.

but i'd be ok to merge this for now and do it "right" in a follow up, so we can get the tests stable.

(I have no merge power in this repo)